CairoMakie/test/runtests.jl

```diff
@@ -134,6 +134,11 @@ end
     rm("test.mp4")
 end
 
+@testset "plotlist no ambiguity" begin
+    plotlist([Makie.SpecApi.Scatter(1:10)])
+    plotlist!([Makie.SpecApi.Scatter(1:10)])
```

Comment:
I'm not 100% sure that this test fully covers this functionality.  Maybe construct or display to a CairoScreen after the first `plotlist` call, to be sure?